`setify` in 08_data_block fails test

I tried running the notebook `08_data_block.ipynb` on Google Colab. For setup i ran this notebook: https://colab.research.google.com/gist/davidpfahler/e9832bd93cff7fffb95d656b56c3197f/setup_fastai_test.ipynb

I then ran each `dl2` folder notebook in the correct order and exported the `.py` files to the same Google Drive directory, which worked up until `08_data_block.ipynb` where I get the following `AssertionError`:

```
AssertionError: ==:
{'a'}
{'aa'}
```
The notebook as run can be accessed [here](https://colab.research.google.com/gist/davidpfahler/ac762c1c43b3ddd62dfc76093188d270/08_data_block.ipynb).

I did see [this answer](https://forums.fast.ai/t/lesson-11-discussion-and-wiki/43406/286?u=davidpfahler) by Jeremy, but `git pull`ing did not help and so I cloned the entire repo from scratch as described above and was able to reproduce the error.

It's not critical to anything else in the course as far as I can tell. but it is really unsettling for beginners that think they do something wrong.

--- a/nbs/dl2/04_callbacks.ipynb
+++ b/nbs/dl2/04_callbacks.ipynb
@@ -528,6 +528,7 @@
     "def listify(o):\n",
     "    if o is None: return []\n",
     "    if isinstance(o, list): return o\n",
+    "    if isinstance(o, str): return [o]\n",
     "    if isinstance(o, Iterable): return list(o)\n",
     "    return [o]"
    ]

